Sources/Clappr_iOS/Classes/Animation/SeekBubble/SeekBubble.swift

```diff
@@ -27,35 +27,37 @@ class SeekBubble: UIView {
     }
     
     private func animateBubble() {
+        guard let parentView = parentView else { return }
+        
         parentView.bringSubviewToFront(self)
         bubbleWidth.constant = parentView.frame.width
         bubbleHeight.constant = parentView.frame.height * 1.8
         
         UIView.animate(withDuration: ClapprAnimationDuration.seekBubbleShow, animations: {
             self.alpha = 1.0
             self.addRoundedBorder(with: self.bubbleHeight.constant / 2)
-            self.parentView.layoutSubviews()
+            self.parentView?.layoutSubviews()
```

Comment:
Do you think it make sense to use the `parentView` reference that we just checked using `guard let` to avoid unnecessary null checks, and do the same in the other methods?

Reply:
It does make sense, thanks for suggestion. 😀